internal/captain/command.go

```diff
@@ -357,6 +358,17 @@ func (c *Command) Flags() []*Flag {
 	return c.flags
 }
 
+func (c *Command) ActiveCommands(args []string) []*Command {
+	var cmds []*Command
+	cmds = append(cmds, c)
+	for _, child := range c.Children() {
+		if funk.Contains(args, child.Name()) {
+			cmds = append(cmds, child)
+		}
+	}
+	return cmds
+}
```

Comment:
This is error prone since you're not respecting the order of the arguments.

Run-command uses `c.UseFull()`, can we use that here as well?

https://github.com/ActiveState/cli/blob/5dc25c28a3dd85bddbfbc22a8fd68f5205025e1b/internal/captain/command.go#L561-L561

Reply:
`c.UseFull()` works from the bottom-up while in this case, we are working top-down. Was still able to get this working but we do have to identify the lowest level child command.